CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required( VERSION 3.0 )
+
+project( cubature )
+
+set( CMAKE_C_FLAGS " -fPIC ${CMAKE_C_FLAGS}" )
```

Comment:
`-fPIC` should be set by default when building a shared library: https://cmake.org/cmake/help/v3.4/prop_tgt/POSITION_INDEPENDENT_CODE.html#prop_tgt:POSITION_INDEPENDENT_CODE